pyccel/codegen/printing/ccode.py

```diff
@@ -586,6 +602,68 @@ def function_signature(self, expr):
         else:
             return '{0}{1}({2})'.format(ret_type, name, arg_code)
 
+    def _print_IndexedElement(self, expr):
+        if isinstance(expr.base, IndexedVariable):
+            base = expr.base.internal_variable
+        else:
+            base = expr.base
+        base = self._print(expr.base.label)
+        inds = list(expr.indices)
+        inds = inds[::-1]
+        base_shape = Shape(expr.base)
```

Comment:
Is there any reason why you chose to use `Shape(expr.base)` here instead of `expr.base.shape`?

Reply:
i thought the shape can be changed so i wanted to use Shape just in case because it return a PythonTuple

Reply:
A tuple can't be changed. If you want to be able to change the shape then you should copy it into a list

Reply:
okey then i think i will change it to `expr.base.shape`